meta-rinato/recipes-kernel/samsung-smartwatch-firmware/samsung-smartwatch-firmware_master.bb

```diff
@@ -0,0 +1,35 @@
+SECTION = "kernel"
+SUMMARY = "Firmware for Samsung rinato smartwatch"
+HOMEPAGE = "https://github.com/casept/linux-samsung-smartwatch-firmware"
+LICENSE = "CLOSED"
+COMPATIBLE_MACHINE = "rinato"
+
+SRC_URI = " git://git@github.com/casept/linux-samsung-smartwatch-firmware.git;protocol=https;branch=master \
+    file://fake-copying"
+
+# For random firmware files extracted from wherever, the licensing is of course unclear.
+# Just create an empty file and checksum that to make yocto shut up.
+LIC_FILES_CHKSUM = "file://fake-copying;md5=d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
Since you have `LICENSE = "CLOSED"` this stuff is not needed. In fact it currently throws a warning:
```
WARNING: samsung-smartwatch-firmware-master-r0 do_populate_lic: Could not copy license file /media/Data/AsteroidOS/asteroid-scarthgap/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/samsung-smartwatch-firmware/master/git/fake-copying
 to /media/Data/AsteroidOS/asteroid-scarthgap/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/samsung-smartwatch-firmware/master/license-destdir/armv7vehf-neon/samsung-smartwatch-firmware/fake-copying: [Errno 2] No such file or direc
tory: '/media/Data/AsteroidOS/asteroid-scarthgap/build/tmp-glibc/work/armv7vehf-neon-oe-linux-gnueabi/samsung-smartwatch-firmware/master/git/fake-copying'
```

Reply:
Removed.